README.md

```diff
@@ -146,4 +146,17 @@ For AOD, `viirs_aod2ioda.py`, is used to convert the native netCDF format for ob
 ```
 Usage: <converter.py> -i INPUT_FILE(S) -o OUTPUT_FILE -m nesdis -k maskout -t 0.0
 ```
-For method option (-m) of bias and uncertainty calculation (default/nesdis), deafult means to set bias and uncertainty as 0.0 and nesdis means to use NESDIS bias and uncertainty calculation method. For maskout option (-k) default/maskout, default means to keep all missing values and maskout means to mask all missing values out. For thinning option, the value should be within 0.0 and 1.0 depending how much data will be thinned, and 0.0 means without any thining.    
+For method option (-m) of bias and uncertainty calculation (default/nesdis), deafult means to set bias and uncertainty as 0.0 and nesdis means to use NESDIS bias and uncertainty calculation method. For maskout option (-k) default/maskout, default means to keep all missing values and maskout means to mask all missing values out. For thinning option, the value should be within 0.0 and 1.0 depending how much data will be thinned, and 0.0 means without any thining.
+
+
+## land
+
+The land converters include all converter scripts for snowpack, soil, vegeation, and the other surface related land variables.
+
+For snow cover fraction(scf), IMS grib2 files are supported with `ims_scf2ioda.py`.
+```
+Usage: ims_scf2ioda.py -i input_ims_file.grib2 -o output_ioda_file.nc -m maskout
+```
+For -i you can specify an input file and the converter will write it to one output file. For maskout option (-m) default/maskout, default means to keep all missing values and maskout means to mask all missing values out.
```

Comment:
Missing vs mask is maybe not the best description, maybe say that maskout means "to not write out missing values"

Reply:
Corrected.